### PR TITLE
Feature/migrate to jhipster 6.5.x

### PR DESCRIPTION
**Pull request by vkirichenko**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -156,30 +148,29 @@ task cucumberTestReport(type: TestReport) {
...
-apply from: 'gradle/docker.gradle'
-apply from: 'gradle/sonar.gradle'
-//jhipster-needle-gradle-apply-from - JHipster will add additional gradle scripts to be applied here
 
-if (project.hasProperty('prod')) {
-    apply from: 'gradle/profile_prod.gradle'
-} else {
-    apply from: 'gradle/profile_dev.gradle'
+checkstyle {
+    toolVersion = "7.6.1"
```
Comment (saylenty) [inline build.gradle]: Why hardcode, it's better to use variable like `"{$checkstyle_tool_version}"` from gradle.properties

File: build.gradle [hunk 2]
```diff
@@ -4,121 +4,113 @@ buildscript {
...
+// See https://virgo47.wordpress.com/2018/09/14/classpath-too-long-with-spring-boot-and-gradle/ for details
+// https://github.com/jhipster/generator-jhipster/issues/9713
 if (OperatingSystem.current().isWindows()) {
-    // https://stackoverflow.com/questions/40037487/the-filename-or-extension-is-too-long-error-using-gradle
     task classpathJar(type: Jar) {
-        dependsOn configurations.runtime
-        appendix = 'classpath'
+        inputs.files sourceSets.main.runtimeClasspath
 
+        archiveName = "runboot-classpath.jar"
```
[the remark this answers is not in the document]
Comment (vkirichenko) [reply build.gradle]: done

File: gradle.properties [hunk 3]
```diff
@@ -2,23 +2,36 @@ rootProject.name=timeline
...
 liquibaseTaskPrefix=liquibase
 
+jaxb_runtime_version=2.3.2
+
+# gradle plugin version
+jib_plugin_version=1.7.0
+git_properties_plugin_version=2.2.0
+apt_plugin_version=0.21
+sonarqube_plugin_version=2.8
+spring_no_http_plugin_version=0.0.3.RELEASE
```
[the remark this answers is not in the document]
Comment (vkirichenko) [reply gradle.properties]: done

File: settings.gradle [hunk 4]
```diff
@@ -1 +1,17 @@
...
+pluginManagement {
+     plugins {
+          id 'org.springframework.boot' version "${spring_boot_version}"
+          id 'com.google.cloud.tools.jib' version "${jib_plugin_version}"
+          id 'com.gorylenko.gradle-git-properties' version "${git_properties_plugin_version}"
+          id 'org.sonarqube' version "${sonarqube_plugin_version}"
+          id 'net.ltgt.apt-eclipse' version "${apt_plugin_version}"
+          id 'net.ltgt.apt-idea' version "${apt_plugin_version}"
+          id 'net.ltgt.apt' version "${apt_plugin_version}"
+          id "io.spring.nohttp" version "${spring_no_http_plugin_version}"
```
[the remark this answers is not in the document]
Comment (vkirichenko) [reply settings.gradle]: done

File: build.gradle [hunk 5]
```diff
@@ -156,30 +148,29 @@ task cucumberTestReport(type: TestReport) {
...
-    apply from: 'gradle/zipkin.gradle'
+spotbugs {
+    ignoreFailures = true
 }
 
-group = 'com.icthh.xm.ms.timeline'
-version = '0.0.1-SNAPSHOT'
+pmd {
+    rulePriority = 3
+    toolVersion = '5.5.4'
```
[the remark this answers is not in the document]
Comment (vkirichenko) [reply build.gradle]: done

File: build.gradle (hunk 1 above)
Comment (vkirichenko) [reply build.gradle]: done

File: src/main/resources/config/application.yml [hunk 6]
```diff
@@ -190,4 +192,5 @@ application:
     cassandra:
         migration-folder: config/cql/changelog
         keyspace-create-cql: "CREATE KEYSPACE IF NOT EXISTS %s WITH REPLICATION = {'class':'SimpleStrategy','replication_factor':3} AND DURABLE_WRITES = true;"
+        keyspace-to-use: "USE %s"
```
Comment (amedvedchuk) [inline src/main/resources/config/application.yml]: @vkirichenko sugest to rename property to `keyspace-switch-cql`

Comment (vkirichenko) [reply src/main/resources/config/application.yml]: done

**Pull request merged**